lagtraj/forcings/conversion/input_definitions_nudging.py

```diff
@@ -0,0 +1,60 @@
+"""
+For nudging parameters we have quite a few sets of parameters and they can be
+used to set the parameters for nudging momentum and scalar forcing profiles
+separately (for example `nudging_timescale_scalars` and
+`nudging_timescale_momentum`). To avoid typos while enumerating all those
+options (and the cases for each nudging method were each parameter is required)
+this code takes care of generating the valid combinations.
+"""
+
+
+AVAILABLE_NUDGING_METHODS = [
+    "fixed_height",
+    "runtime_inversion_height",
+    "constant",
+]
+
+NUDGING_PARAMS_VALID_VALUES = dict(
+    height=float, layer_thickness=float, timescale=int, shape=["cos"]
+)
+
+
+def _construct_nudging_required_fields():
+    """
+    parameters for nudging scalars and momentum are set separately, so we add
+    them here in turn to the set of required parameters
+    """
+    fields = {}
+    for nudged_variable_groups in ["scalars", "momentum"]:
+        fields[
```

Comment:
Is there a more informative variable name here?

Reply:
Rename to `nudging_parameters` thank you!